typescript/lib/bulk_command_execution/v0_bulk_command_api/v0_bulk_commands_test.ts

```diff
@@ -0,0 +1,41 @@
+import { RegisterServiceArgs } from "../../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { V0SerializableCommand } from "./v0_bulk_commands"
+import { describe } from "mocha";
+import { expect } from "chai";
+
+
+describe ('test deserializeRegisterServiceJSON()', () => { //TODO - what exactly are we testing, I believe we are testing if V0SerializableCommand is set up appropriately?
+	it ('should check appropriate instantiation of V0SerializableCommand', () => {
+    
+        var deserialized: V0SerializableCommand; // TOOD - REMOVE = new V0SerializableCommand();
+        const serviceId: string = "my-service-id";
+        const jsonStr: string = '`"{"type":"REGISTER_SERVICE", "args":{"service_id":'+ serviceId + '}}`)';
```

Comment:
Does this test pass? AFAIK this JSON string will render to `{"type":"REGISTER_SERVICE", "args":{"service_id":my-service-id}}` which isn't valid JSOn because `my-service-id` isn't wrapped in `"`